resources/resincli.robot

```diff
@@ -95,6 +97,18 @@ Remove ENV variable ${variable_name} from application ${application_name}
     ${result} =  Run Process    resin env rm ${result_id.stdout} --yes     shell=yes
     Process ${result}
 
+Get public address of device ${device_uuid}
+    Run Process    resin device public-url enable ${device_uuid}    shell=yes
+    ${result} =  Run Process    resin device public-url ${device_uuid}    shell=yes
+    Process ${result}
+    Return From Keyword    ${result.stdout}
+
+Make the request to ${address}
```

Comment:
Noted, will make generic keywords from now on and a separate Robot file for dealing with this HTTP request and for the ones to come.